ui/src/layout/SubMenu.js

```diff
@@ -30,19 +37,37 @@ const SubMenu = ({
   icon,
   children,
   dense,
+  secondaryLink = '',
+  secondaryAction,
 }) => {
   const translate = useTranslate()
   const classes = useStyles()
 
   const header = (
-    <MenuItem dense={dense} button onClick={handleToggle}>
-      <ListItemIcon className={classes.icon}>
-        {isOpen ? <ExpandMore /> : icon}
-      </ListItemIcon>
-      <Typography variant="inherit" color="textSecondary">
-        {translate(name)}
-      </Typography>
-    </MenuItem>
+    <div className={classes.headerWrapper}>
+      <MenuItem
+        dense={dense}
+        onClick={handleToggle}
+        className={classes.menuHeader}
+      >
+        <ListItemIcon className={classes.icon}>
+          {isOpen ? <ExpandMore /> : icon}
+        </ListItemIcon>
+        <Typography variant="inherit" color="textSecondary">
+          {translate(name)}
+        </Typography>
+      </MenuItem>
+      {secondaryLink && sidebarIsOpen ? (
+        <MenuItemLink
+          to={secondaryLink}
+          primaryText={<ArrowForwardIcon />}
```

Comment:
I like the implementation, but the Array is too big. Can we make something more subtle? Maybe a smaller, hidden button that only shows on hover? Maybe a less "aggressive" icon, like `ChevronRight`?

Reply:
If we make the button appear only on hover, then mobile users will not be able to navigate to the playlists page right..?

I have changed the button to RiEdit2Line. Tho personally, I feel a right arrow was best at giving out the meaning of going into the list of playlists. The RiEdit2Line IMHO gives out the feeling that this page is for edits, whereas it could be used by many people for getting details, performing batch actions on playlists etc..

Reply:
> If we make the button appear only on hover, then mobile users will not be able to navigate to the playlists page right..?

This is not a show stopper of course, but on smaller screens you can force it to always show. We already do it with the context menu, and the Album cover's `GridListTileBar` for example.

Yeah, some kind of arrow would be better, but I think that one is too "bold". Can you try with some other arrow (maybe `IoIosArrowDropright`), and set its size to "small"?